### ISSUE TITLE
Criar code of conduct.md

### ISSUE THREAD

**Issue by gab0gomes** [Gabrielr2508/template-tcc-latex-univasf#17]
Essa issue tem a intenção de adicionar ao projeto o arquivo `CODE_OF_CONDUCT.md` para estabelecer o comportamento esperado dos participantes do projeto. Ajudando a criar uma atmosfera social positiva para a comunidade.

Um bom exemplo é o arquivo do [Open Source Guides](https://github.com/Gabrielr2508/opensource.guide/blob/master/CODE_OF_CONDUCT.md). 

Se você tiver alguma sugestão de código de conduta, nos avise, por favor. :)

Não deixe de perguntar se tiver alguma dúvida.

**Comment by RaulLima2** [comment]
Olá, Gabriel, como seria sua proposta de "code of conduct.md"?

**Comment by gab0gomes** [comment]
Oi, @RaulLima2.

Como mencionado na descrição da issue, o CODE_OF_CONDUCT do opensource.guides é um bom começo. Acredito que possa ser o mesmo código, com as devidas alterações para o escopo desse projeto.

**Comment by RaulLima2** [comment]
é possível adequar algumas partes para o projeto, porém deve ter nesta parte a forma como vai ser escrito (o padrão) e as  questões de erros ortográficos no material.

**Comment by gab0gomes** [comment]
Desculpa, @RaulLima2, mas não entendi o que você quis dizer. :(

**Comment by RaulLima2** [comment]
Ok, então, percebi que você vai escrever um Código de Conduta para o template-tcc-latex-univasf, seria, um ótima ideia adequar algumas partes do "code of conduct", porém, deve ter a forma como vai ser escrito (o padrão exigido) e a resposabilidade do contribuinte na ortografia, para que não haja erro.

**Comment by gab0gomes** [comment]
Você diz o padrão de escrita e ortografia no arquivo de código de conduta ou nas contribuições em si?

**Comment by RaulLima2** [comment]
no código de conduta,

**Comment by gab0gomes** [comment]
Desculpa, mas continuo sem entender o propósito de colocar isso no código de conduta. Acredito que essa questão de ortografia e normas de escrita devam estar contidas no arquivo `CONTRIBUTING.md`. Ao meu ver o código de conduta diz respeito à como as pessoas devem se portar dentro da comunidade, seja virtualmente ou em encontros presenciais.

Você enxerga diferente?

**Comment by RaulLima2** [comment]
Sim, e verdade, logo, está questão não cabe neste escopo. Portanto, acho que não mudaria muito o código de conduta de como as pessoas devem se comporta dentro da cominidade, já que está dentro do que é proposto.
